test/unit/extract/excel_test.py

```diff
@@ -1,35 +1,42 @@
 """Unit tests for pudl.extract.excel module."""
 
-import unittest
-from unittest import mock as mock
-
 import pandas as pd
+import pytest
 
 from pudl.extract import excel
 
 
-class TestMetadata(unittest.TestCase):
+class TestMetadata:
     """Tests basic operation of the excel.Metadata object."""
 
+    @pytest.fixture(autouse=True)
```

Comment:
This works, but the recommended way of implementing these xunit-style setup methods is to use `def setup_class`: https://docs.pytest.org/en/7.1.x/how-to/xunit_setup.html#class-level-setup-teardown

But, the even more recommended thing is to just turn this into a normal fixture called `metadata` and refer to that everywhere we need it.